package.json

```diff
@@ -21,26 +25,38 @@
     "webhook": "./packages/cli/bin/n8n webhook",
     "worker": "./packages/cli/bin/n8n worker"
   },
+  "dependencies": {
+    "n8n": "*"
+  },
   "devDependencies": {
+    "@n8n_io/eslint-config": "*",
     "@types/jest": "^28.1.8",
+    "@types/node": "^16.11.22",
     "jest": "^28.1.3",
     "jest-environment-jsdom": "^28.1.3",
     "jest-mock": "^28.1.3",
-    "patch-package": "^6.4.7",
+    "nodemon": "^2.0.2",
```

Comment:
`nodemon` is only used in `cli`. Does it belong here?

Reply:
I'm trying to explicitly "hoist" `devDependencies` whenever it makes sense to me. But, I can move this one back.